package.json

```diff
@@ -31,31 +31,31 @@
   },
   "homepage": "https://github.com/yext/answers-hitchhiker-theme#readme",
   "devDependencies": {
```

Comment:
do we also need to make any updates to the `static/package.json`?

Reply:
I just meant that if you're addressing Snyk issues in the repo, maybe there were also Snyk issues in the `static/package-lock` that would be nice to fix too

Reply:
Ah I see, no the list I went through just had those packages as immediate dependencies with patch fixes

Reply:
oh they're listed separately in Snyk. [here](https://app.snyk.io/org/answers/project/5fa047db-fb78-43cd-9ba1-6c56fbd123bc)'s the project for `static/package.json`

Reply:
updated `resolve-url-loader` and tried `npm run audit`